prac_07/convert_miles_km.py

```diff
@@ -0,0 +1,42 @@
+"""
+CP1404 Practical
+GUI program to convert miles to kilometres
+"""
+
+from kivy.app import App
+from kivy.lang import Builder
+
+
+__author__ = 'Dea Roys'
+
+MILES_TO_KM = 1.60934
+
+class Miles_To_Km(App):
+
+    def build(self):
+        self.title = "Convert miles to kilometres"
+        self.root = Builder.load_file('convert_miles_km.kv')
+        return self.root
+
+    def check_inputs(self):
+        try:
+            user_input = float(self.root.ids.input_number.text)
+            return user_input
+        except ValueError:
+            return 0
```

Comment:
Does this method need _2 return_ statements or could a single variable be changed? Could I _pass_ this method a different input for a different use case? What does this method **check_inputs** for?